web/src/client/storage.js

```diff
@@ -46,6 +46,17 @@ const ZFCP_CONTROLLER_IFACE = "org.opensuse.Agama.Storage1.ZFCP.Controller";
 const ZFCP_DISKS_NAMESPACE = "/org/opensuse/Agama/Storage1/zfcp_disks";
 const ZFCP_DISK_IFACE = "org.opensuse.Agama.Storage1.ZFCP.Disk";
 
+/**
+ * Enum for the encryption method values
+ *
+ * @readonly
+ * @enum { string }
+ */
+const EncryptionMethods = Object.freeze({
+  LUKS2: "luks2",
+  TPM: "tpm_fde"
+});
+
```

Comment:
As I wrote in the commit message, following here the same approac as the current network code. We could, however, goes for what we did with _client/phase_ or _client/status_ instead and move it to a _client/storage-encryption-methods.js_ or _client/storage/encryption-methods.js_ 

```js
export const LUKS2 = "luks2";
export const TPM = "tpm_fde";

export default {
  LUKS2,
  TPM
};
```
and then import needed values 

```js
import { TPM } from "~/client/storage/encryption-methods"
``` 

```js
import * as EncriptionMethods from "~/client/storage/encryption-methods"
```

No strong/clear opinion. Something to think about for future enhancements.